my-project/src/components/Field/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './style.scss';
+
+const myInput = (props) => {
+    const { input, type, placeholder, meta } = props;
+    return (
+        <>
+            <input {...props.input} type={props.type} placeholder={props.placeholder} />
```

Comment:
There these props in const input/type/placeholder, use them here.

Reply:
done